engine/admin/logging/gelf.md

```diff
@@ -52,15 +69,15 @@ $ docker run \
-| `gelf-compression-level` | The level of compression when `gzip` or `zlib` is the `gelf-compression-type`. An integer in the range of `-1` to `9` (BestCompression). Default value is 1 (BestSpeed). Higher levels provide more compression at lower speed.| `--log-opt gelf-compression-level=2` |
-| `tag`                | A string that is appended to the `APP-NAME` in the `gelf` message. By default, Docker uses the first 12 characters of the container ID to tag log messages. Refer to the [log tag option documentation](log_tags.md) for customizing the log tag format. | `--log-opt tag=mailer` |
-| `labels`             | Applies when starting the Docker daemon. A comma-separated list of logging-related labels this daemon will accept. Adds additional key on the `extra` fields, prefixed by an underscore (`_`). Used for advanced [log tag options](log_tags.md).| `--log-opt labels=production_status,geo` |
-| `env`                | Applies when starting the Docker daemon. A comma-separated list of logging-related environment variables this daemon will accept. Adds additional key on the `extra` fields, prefixed by an underscore (`_`). Used for advanced [log tag options](log_tags.md). | `--log-opt env=os,customer` |
-
+| Option                   | Description                                                                                                                                                                                                                                                     | Example value                                     |
+|:-------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------|
+| `gelf-address`           | The address of the GELF server. `udp` is the only supported URI specifier and you must specify the port.                                                                                                                                                        | `--log-opt gelf-address=udp://192.168.0.42:12201` |
+| `gelf-compression-type`  | The type of compression the GELF driver uses to compress each log message. Allowed values are `gzip`, `zlib` and `none`. The default is `gzip`.                                                                                                                 | `--log-opt gelf-compression-type=gzip`            |
+| `gelf-compression-level` | The level of compression when `gzip` or `zlib` is the `gelf-compression-type`. An integer in the range of `-1` to `9` (BestCompression). Default value is 1 (BestSpeed). Higher levels provide more compression at lower speed.                                 | `--log-opt gelf-compression-level=2`              |
```

Comment:
`-1` or `0` is "disable compression" (but not documented here); I  know some people ran into issues with compression (used a lot of resources on high-traffic logging), so may be worth mentioning somewhere